RAMP/unpacker.py

```diff
@@ -27,14 +26,18 @@ def __str__(self):
 
 
 def unpack(bundle, max_bundle_size_mb=None):
-    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes]]
+    # type: (IO[bytes], Optional[int]) -> Tuple[Dict[str, Any], IO[bytes], Dict[str, Any]]
     """
     Unpacks a bundled module, performs sanity validation on bundle.
     both the module metadata and the actual module are returned
     :rtype: tuple
     """
+    deps_files = dict()
     try:
         with ZipFile(bundle) as zf:
+            for filename in zf.namelist():
+                if filename.startswith("deps"):
```

Comment:
what if in the future we will come up with a new module called `depsology`

is this because you expect this to be inside a "deps" directory?
if so, I'd refine this condition

at a minimum `.startswith("deps/")` (if it works)

Reply:
the name of the file is module.so not depsology.so... so its not an issue. but I don't mind adding '/'